pkgs/by-name/ni/nixos-init/README.md

```diff
@@ -0,0 +1,66 @@
+# nixos-init
+
+A system for the initialization of NixOS.
+
+The most important task of nixos-init is to work around the constraints of the
+Filesystem Hierarchy Standard (FHS) that are imposed by other tools (most
+importantly systemd itself).
+
+The primary design principle is to do the minimal work requird to start
+systemd. Everything that can be done later SHOULD be done later (i.e. after
+systemd has already started). This isn't controversial either, this is the
+basic principle behind initrds in the first place.
+
+Extending this init should be done with care and only when strictly necessary.
+Extending it should always be a last resort. It is explitly not designed to be
+extended for third party extensions. I would go so far that all third party
+extensions should be forbidded.
```

Comment:
This is just a piece of rust code, I wouldn't even know what an "extension", be it third-party or internal would mean. I'd probably rephrase the paragraph, stating adding more functionality to it needs good reason, and it's not meant to be extended dynamically?